cmd/cdi-importer/importer.go

```diff
@@ -179,6 +180,13 @@ func handleImport(
 	err := processor.ProcessData()
 
 	scratchSpaceRequired := errors.Is(err, importer.ErrRequiresScratchSpace)
+
+	ipfErr := &importer.ErrImagePullFailed{}
+	isImagePullFailure := errors.As(err, &ipfErr)
+	if isImagePullFailure {
```

Comment:
Why not just do
```go
if errors.As(err, &ipfErr) {
  err = ipfErr
}
```
Also if the `As` is true, isn't err already the right type? I am not entirely I understand what is going on here.

Reply:
We don't need the boolean here. leftover from another implementation during the development. Will fix.

The `As` function does not change `err` but only populate `ipfErr`

Reply:
In 2nd thought, I don't need this code at all. I was trying to write a new field in the `TerminationMessage` object, but this is not working anyway for other reasons. 

I'm now relaying on the error text, that is still there.

Reply:
removed this code